lib/nationbuilder/client.rb

```diff
@@ -56,6 +56,9 @@ def raw_call(path, method, body = {}, args = {})
       request_args[:query].merge!(body)
     else
       body[:access_token] = @api_key
+      unless body[:fire_webhooks].nil?
```

Comment:
If `body[:fire_webhooks] ` is `false`, we won't enter the block of code. `if body[:fire_webhooks]` covers all truthy and falsey values.